web/packages/teleterm/src/ui/AccessRequestCheckout/AccessRequestCheckout.tsx

```diff
@@ -161,6 +161,9 @@ export function AccessRequestCheckout() {
                         break;
                       case 'role':
                         break;
+                      case 'saml_idp_service_provider':
```

Comment:
Instead of adding a separate `case` here, we can combine it with `app`.
So in the line 150. we would have:
```ts
                      case 'app':
                      case 'saml_idp_service_provider':
                        resource.Icon = Icon.Application;
                        break;
```